packages/graphql-transformers-e2e-tests/src/testUtil.ts

```diff
@@ -0,0 +1,56 @@
+import { 
+    ObjectTypeDefinitionNode, FieldDefinitionNode, DocumentNode,
+    InputObjectTypeDefinitionNode, Kind, InputValueDefinitionNode,
+    DefinitionNode
+} from 'graphql';
+import { isNonNullType } from 'graphql-transformer-common';
+
+export function expectFields(type: ObjectTypeDefinitionNode, fields: string[]) {
```

Comment:
Extending the matcher would be a better approach. More on this at [jest docs](https://jestjs.io/docs/en/expect.html#expectextendmatchers)